cogs/startup.py

```diff
@@ -0,0 +1,177 @@
+import os
+import platform
+import random
+from typing import Sequence
+
+import disnake
+import exceptions
+import services.portal.transmission as Transmission
+from disnake import (ApplicationCommandInteraction, Message,
+                     RawMessageDeleteEvent, RawMessageUpdateEvent)
+from disnake.ext import commands, tasks
+from disnake.ext.commands import Bot
+
+
+class Startup(commands.Cog):
```

Comment:
Maybe we could rename this to eventlisteners since this mainly contains event listeners.

Reply:
Cogs group event listeners, commands, etc.

I don't think it makes sense to have an entire Cog grouping all of the listeners in our program. If we want to just group it this way for now, we could get rid of the Cog and simply have it be an extension filled with listeners.

I think that we can split up the different listeners into categories (setup related  ones [on_ready, ...], message related ones, etc.) :+1:

Watcha think?

Reply:
Yeah, that makes sense since we have listeners for setup, messages and errors. So we can definitely split them since they have different roles.

Reply:
Is it alright if we accept this PR now and I'll make a new branch/issue pair for cleaning up the cogs? I don't think it relates to cleaning up `main.py` that much.

I don't want to bundle it all up in one big push (for traceability sake). :+1:

Reply:
Sure, sounds good